migrations/config/intermediate_db.yml

```diff
@@ -113,6 +113,22 @@ schema:
           - "public_topic_count"
           - "staff_topic_count"
           - "target_tag_id"
+    user_custom_fields:
+      columns:
+        exclude:
+          - "id"
+        add:
+          - name: "is_multiselect_field"
+            datatype: boolean
```

Comment:
Is the `is_multiselect_field` column really needed for `user_custom_fields`? I don't see it used anywhere. I think it's only useful for `user_field_values`.